source/Orchestrator/test/test_check_ssm_execution.py

```diff
@@ -348,3 +348,61 @@ def test_execid_parsing_sharr(mocker):
 
     ssmc_stub.deactivate()
     ssmc_stub.deactivate()
+
+def test_missing_account_id(mocker):
+    """
+    Verifies that system exit occurs when an account ID is missing from event
+    """
+    ssm_c = boto3.client('ssm')
+    test_event['SSMExecution']['ExecId'] = '5f12697a-70a5-4a64-83e6-b7d429ec2b17'
```

Comment:
Are all of these tests sharing and modifying the same object? That's concerning. No need to change anything for now.

Reply:
Yeah, it can definitely use an overhaul.